src/common/Common.ts

```diff
@@ -142,18 +129,16 @@ export class CommandLineUtils {
                         validate: CommandLineUtils.validateApiLevelFlag
                     })
                 };
-            case FlagsConfigType.Platform:
+            case FlagsConfigType.PlatformType:
                 return {
-                    platform: Flags.string({
+                    platform: Flags.option({
                         char: 'p',
-                        description: supportsDesktop
-                            ? messages.getMessage('platformFlagIncludingDesktopDescription')
-                            : messages.getMessage('platformFlagMobileOnlyDescription'),
-                        required: isRequired,
-                        validate: supportsDesktop
-                            ? CommandLineUtils.validatePlatformFlagIncludingDesktop
-                            : CommandLineUtils.validatePlatformFlagMobileOnly
-                    })
```

Comment:
No need to define a custom message. For flags of type `option` the help message automatically adds the list of available options so makes our code simpler